cudaaligner/tests/Test_MyersAlgorithm.cu

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
Not sure if git got confused here, but just in case you did not know in situation like this you can use `git mv cudaaligner/tests/Test_MyersAlgorithm.cpp cudaaligner/tests/Test_MyersAlgorithm.cu`.
Sometimes if the renamed file changed too much git still decides to treat them as deletion + addition instead of rename

Reply:
I used `git mv`, but somehow git got confused. Probably because I added much code afterwards in the same commit.